protocol/messenger.go

```diff
@@ -4110,6 +4110,32 @@ func (m *Messenger) MarkMessagesSeen(chatID string, ids []string) (uint64, uint6
 	return count, countWithMentions, nil
```

Comment:
@ilmotta @qfrank if its possible to replace `count` and `countWithMentions` with affected messages in the response i would like to change response type to `MessengerResponse`

Reply:
Basically, I only need AC notifications in the response, so I'm fine with the current changes. However, using a regular MessengerResponse seems cleaner to me. Can do a separate one